middleware/TelemetryHandler.md

```diff
@@ -6,9 +6,30 @@ Provide a middleware component that attaches metadata to a Graph request in orde
 
 ## Requirements
 
-- Add `SdkVersion` request header to each request to identify the language and version of the client SDK library.  The header value should be of the format `graph-{lang}-{version}`.
-- Add a `client-request-id` header with GUID value to request if one is not present.  If the [RequestContext](../middleware/middleware.md) has a `ClientRequestId` property set then that value should be used.
-- Add a `FeatureUsage` header using the value of the FeatureUsage property on RequestContext.  
+- Add a `client-request-id` header with GUID value to request if one is not present.  If the [RequestContext](../middleware/RequestContext.md) has a `ClientRequestId` property set then that value should be used.
+- Add `SdkVersion` request header to each request to identify the language and version of the client SDK library.  The product identifier should be of the format `graph-{lang}`. The `SdkVersion` version can contain multiple values, or appear multiple times in a request. Values should be comma delimited as normal header lists are. e.g.
+
+    `SdkVersion: MyApp/1.0, spfx/1.0, graph-javascript-1.0`
+- If available, add a the `featureUsage` value contained in the [RequestContext](../middleware/RequestContext.md) as key/value pair in the comment after the product. e.g.
+
+    `SdkVersion: graph-dotnet/1.0 (featureUsage=0f)`
+
```

Comment:
- Higher layer libraries that depend on our core library MUST add an SdkVersion header value. Service libraries SdkVersion headers should take the form `graph-{lang}-{endpoint}/{version}`.